src/main/java/frc/robot/subsystems/shooter/ShooterFlywheelIOSparkMax.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.subsystems.shooter;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import frc.robot.Constants;
+
+public class ShooterFlywheelIOSparkMax implements ShooterFlywheelIO
+{
+    private CANSparkMax     _upperFlywheelSparkMax;
+    private CANSparkMax     _lowerFlywheelSparkMax;
+    private RelativeEncoder _upperFlywheelEncoder;
+    private RelativeEncoder _lowerFlywheelEncoder;
+
+    public ShooterFlywheelIOSparkMax()
+    {
+        _upperFlywheelSparkMax = new CANSparkMax(Constants.CAN.SHOOTER_FLYWHEEL_UPPER, MotorType.kBrushless);
+        _lowerFlywheelSparkMax = new CANSparkMax(Constants.CAN.SHOOTER_FLYWHEEL_LOWER, MotorType.kBrushless);
+        _lowerFlywheelSparkMax.setInverted(true); // FIXME: Change if not correct
+
+        _upperFlywheelEncoder = _upperFlywheelSparkMax.getEncoder();
+        _lowerFlywheelEncoder = _lowerFlywheelSparkMax.getEncoder();
+    }
```

Comment:
Get references to the built-in PID controllers, then set their coefficients:

```suggestion
        _upperPIDController = _upperFlywheelSparkMax.getPIDController();
        _lowerPIDController = _lowerFlywheelSparkMax.getPIDController();
        
        // Find proper coefficients
        for (var pid : new SparkPIDController[] { _upperPIDController, _lowerPIDController })
        {
            pid.setP(6e-5);
            pid.setI(0);
            pid.setD(0);
            pid.setIZone(0);
            pid.setFF(0.000015);
            pid.setOutputRange(-1, 1);
        }
    }
```